### PR TITLE
Fix `open-all-conversations` feature

### LINKED ISSUE #5679: `open-all-conversations` doesn't appear when selecting conversations
### Example URLs

https://github.com/refined-github/refined-github/issues
https://github.com/refined-github/refined-github/pulls

### Extension version

22.5.22

### Browser(s) used

Safari, Chrome

### Description

1. Click "Open all"
2. Nothing happens

Alternatively:

1. Check 1 or more files
2. No "Open selected" appears

https://user-images.githubusercontent.com/1402241/171992568-7b3e8b7f-d0b6-4e14-bd7a-22b018eb0243.mov

### PR DESCRIPTION
**Pull request by cheap-glitch**
Fix #5679 

## Test URLs

* [Repo PR list](https://github.com/refined-github/refined-github/pulls?q=is%3Apr+is%3Aopen+sort%3Aupdated-desc)
* [Global PR list](https://github.com/pulls?q=is%3Apr+is%3Aopen+author%3A%40me+archived%3Afalse+sort%3Aupdated-desc)

## Screenshot

https://user-images.githubusercontent.com/46634000/172637096-85b9f6fa-4aca-46b5-b1d4-634d25047e72.mp4

### CODE REVIEW

File: source/features/open-all-conversations.tsx [hunk 1]
```diff
@@ -13,15 +14,12 @@ function getUrlFromItem(issue: Element): string {
...
-export function confirmOpen(count: number): boolean {
-	return count < 10 || confirm(`This will open ${count} new tabs. Continue?`);
-}
+const issueListSelector = pageDetect.isGlobalConversationList()
+	? '#js-issues-toolbar div'
+	: 'div[aria-label="Issues"][role="group"]';
 
 function onButtonClick(): void {
-	const modifier = pageDetect.isGlobalConversationList() ? '' : ' + div ';
-	const issues = select.all(`#js-issues-toolbar:not(.triage-mode) ${modifier} .js-issue-row`);
```
Comment (cheap-glitch) [inline source/features/open-all-conversations.tsx:23]: The `.triage-mode` part of the selector was a vestigial residue from the previous incarnation of the feature (`batch-open-conversations`). Now we don't really care whether some issues are checked or not.

Comment (fregante) [reply source/features/open-all-conversations.tsx:23]: Me again 2 minutes ago https://github.com/refined-github/refined-github/issues/4085#issuecomment-949492199 😂

File: source/features/open-all-conversations.tsx [hunk 2]
```diff
@@ -13,15 +14,12 @@ function getUrlFromItem(issue: Element): string {
 		.href;
 }
 
-// eslint-disable-next-line import/prefer-default-export
-export function confirmOpen(count: number): boolean {
```
Comment (fregante) [inline source/features/open-all-conversations.tsx:17]: I'd include actually opening the tabs as part of this utility, like `openTabs(urls: string[])`

In `open-all-notifications` we can do the other DOM manipulation _after_ we open the tabs

Comment (fregante) [reply source/features/open-all-conversations.tsx:17]: Not necessary as part of this PR though, you can merge it at will

File: source/features/open-all-notifications.tsx [hunk 3]
```diff
@@ -6,30 +6,30 @@ import * as pageDetect from 'github-url-detection';
...
 
 function openNotifications(notifications: Element[], markAsDone = false): void {
-	// Ask for confirmation
-	if (!confirmOpen(notifications.length)) {
+	const urls: string[] = [];
+	for (const notification of notifications) {
+		urls.push(notification.querySelector('a')!.href);
+	}
+
+	if (!openTabs(urls)) {
```
Comment (FloEdelmann) [inline source/features/open-all-notifications.tsx:21]:
```suggestion
	const urls = notifications.map(notification => notification.querySelector('a')!.href);
	if (!openTabs(urls)) {
```

**Review by FloEdelmann — COMMENTED** [review]
Sorry, I was a little too late. I've got one code style suggestion, maybe that can be done in the next Lint PR.

**Pull request merged**